hicommonwealth/commonwealth
# [ERC20 Launchpad] - Added success step for tokenized community flow

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9321

## Description of Changes
Added success step for tokenized community flow

## "How We Fixed It"
N/A

## Test Plan

- Enable `FLAG_TOKENIZED_COMMUNITY`
- Visit `/createTokenCommunity`
- Verify you see the success step of the tokenized community creation flow.

## Deployment Plan
N/A

## Other Considerations
N/A
